providers/hypersh/hypersh.go

```diff
@@ -219,7 +219,7 @@ func (p *HyperProvider) CreatePod(ctx context.Context, pod *v1.Pod) error {
 	// Iterate over the containers to create and start them.
 	for k, ctr := range containers {
 		//one container in a Pod in hyper.sh currently
-		containerName := fmt.Sprintf("pod-%s-%s", pod.Name, pod.Spec.Containers[k].Name)
+		containerName := containerName(pod.Namespace, pod.Name)
```

Comment:
don't quite understand the logic here. do all the containers in a pod use the same container name in hyper?